prow/cmd/peribolos/main.go

```diff
@@ -415,6 +415,7 @@ func (m *memberships) normalize() {
 func configureMembers(have, want memberships, invitees sets.String, adder func(user string, super bool) error, remover func(user string) error) error {
 	have.normalize()
 	want.normalize()
+	invitees = normalize(invitees)
```

Comment:
Maybe I'm missing something here...but shouldn't we normalize invitees in `configureOrgMembers` so that the following in 
`configureOrgMembers` will yield the correct result?

https://github.com/kubernetes/test-infra/blob/c0628c303df03609b78b823aa65149e4a125e39a/prow/cmd/peribolos/main.go#L363

The normalized `invitees` can then be passed onto `configureMembers`, instead of normalizing here.